src/server/index.spec.tsx

```diff
@@ -9,6 +9,8 @@ import { RequestWithCookies } from 'universal-cookie-express';
 import Cookies from 'universal-cookie';
```

Comment:
Is this a test file?  How it is we're expected to differentiate between tests and live code?

Reply:
In code-manager, files with `.spec` in their filename are test files. We generally name the tests the same as the source files, with `.spec`, so in this case `src/server/index.tsx` and `src/server/index.spec.tsx`.